scripts/bin/hugo

```diff
@@ -1,19 +1,5 @@
 #!/usr/bin/env bash
 
-# Copyright 2019 The Jetstack cert-manager contributors.
```

Comment:
Why has all this been removed?

Reply:
I'm not certain, but it seems you may have misinterpreted the linked CNCF guideline:

```
CNCF does not require or recommend that every contributor include their copyright notice in contributed files. See below for more details on why not.
```

From my reading, this implies that _every_ contributor does not need to list the *names* of copyright holders, in favour of having a general statement of "the cert-manager contributors". (rather than saying that we don't need it in every _file_).

I think it's good to keep here, as automated license checkers can also pick this stuff up (which others could be making use of)

Reply:
You are right, I definitely misinterpreted the guidelines 😞 I'll re-add the license notice

Shouldn't we remove the mention of Jetstack though? e.g.:

```diff
-# Copyright 2019 The Jetstack cert-manager contributors.
+# Copyright 2019 The cert-manager contributors.
```

Reply:
Out of curiosity, why do we have to add the license notice to every single file; is it legally necessary?

Reply:
Yes we should remove the Jetstack part 😄

> Out of curiosity, why do we have to add the license notice to every single file; is it legally necessary?

If someone else copies our files into their own project, they'll need to add this header themselves (and often may not/may forget). Also, if there are multiple licenses in the repository (e.g. our logo is licensed differently to code), it is explicit and clear what license is in use for that content.

That's my thinking anyway! :)